tutorials/basic_1_square/README.md

```diff
@@ -0,0 +1,137 @@
+# Basic: Rendering a simple square on the screen
+
+This tutorial will introduce you to:
+
+ - `Game`: The basic class to build a game on, you will understand its structure and how to use it to build a simple game
+ - `GameWidget`: The widget used to place your game on your widget tree
+ - Basic rendering: You will get introduced to the basics of using `dart:ui` class `Canvas` which is the class which Flame uses for rendering your game.
+
+All the code of this tutorial code can be found [here](./code) and is based on Flame `1.0.0-rc7` version.
+
+By the end of this tutorial you will have built a simple game that renders a square bouncing on the screen that will look like this:
+
+![Preview](./media/preview.gif)
+
+## Building your first game
+
+`Game` is the most basic class that you can use to build your game, it includes the necessary methods for creating a basic game loop and methods for the lifecycle of a Flame game.
```

Comment:
Agreed, even tho the `Game` class is indeed the most basic class, it is not the easiest of them to use when you are just learning Flame. As most of our own documentation is more focused on both FCS and `BaseGame`.

`Game` is a more a low-level API class imho

Reply:
But a counter to that would be that setting up FCS without basic understanding of our gameloop/logic is also counter-intuitive.. So I can go either way on this

Reply:
I think we should approach Game before approaching BaseGame. I intentionally choose Game here, because I think I think it is important that people know that there are both classes and why they exist and most important, I think that for this tutorial, which introduces what a game loop is, this is way more fit than BaseGame.

I intent to do more tutorials afterwards and basically all of them will use BaseGame instead of Game, just these firsts one I will use Game.

Reply:
I would at least comment: normally you will use BaseGame that contains a lot of utilities but we will see that on tutorial xxx

Reply:
I agree that a comment about BaseGame would be good too, I don't think the user will drift away because of that.

Reply:
Alright, added a comment about BaseGame, let me know if that is fine